cirq/linalg/transformations_test.py

```diff
@@ -168,6 +168,31 @@ def test_targeted_left_multiply_out():
         atol=1e-8)
 
 
+def test_targeted_conjugate():
+    a = np.reshape([0, 1, 2j, 3j], (2, 2))
+    b = np.reshape(np.arange(16), (2,) * 4)
+    result = cirq.targeted_conjugate(a, b, [0], [2])
+    expected = np.einsum('ij,jklm,ln->iknm', a, b,
+                         np.transpose(np.conjugate(a)))
+    np.testing.assert_almost_equal(result, expected)
+
+    result = cirq.targeted_conjugate(a, b, [1], [3])
+    expected = np.einsum('ij,kjlm,mn->kiln', a, b,
```

Comment:
Make a dead simple test that has raw numbers, so we're not relying on having correctly written the einsum expression.

Reply:
done